src/main/java/com/metamx/http/client/HttpClientConfig.java

```diff
@@ -67,6 +67,8 @@ public String getEncodingString()
   // Default from SelectorUtil.DEFAULT_IO_THREADS, which is private:
   private static final int DEFAULT_WORKER_COUNT = Runtime.getRuntime().availableProcessors() * 2;
 
+  private static final long DEFAULT_UNUSED_CONNECTION_TIMEOUT = 4*60*60*1000*1000l;
```

Comment:
can you use Duration for the type and also we don't need nano seconds for this config.

Reply:
1) Please add `_MICROS` suffix (intended microseconds, right?)
2) Use `TimeUnit.HOURS.toMicros(4)`